catalyst/dl/runner/gan.py

```diff
@@ -2,8 +2,9 @@
     Any, Callable, Dict, List, Mapping, Optional, Tuple, Union
 )
 
-from catalyst.dl import Runner
-from catalyst.utils.tools.typing import Device, Model
+from catalyst.dl import GanExperiment, Runner
+from catalyst.dl.state import GanState
```

Comment:
why do you need such import? looks like `from catalyst.dl import GanExperiment, GanState , Runner` should be enough

Reply:
Done